plugins/aws-costs/README.md

```diff
@@ -1,3 +1,28 @@
 # aws-costs
 
-Queries and saves [AWS costs and Usage Report](https://docs.aws.amazon.com/cur/latest/userguide/what-is-cur.html) given a cluster.
\ No newline at end of file
+AWS costs plugin is built on [AWS costs and Usage Report](https://docs.aws.amazon.com/cur/latest/userguide/what-is-cur.html).
```

Comment:
Thanks James! Can we actually move this to https://github.com/FairwindsOps/insights-docs/blob/main/docs/technical-details/reports/aws-costs.md

Reply:
alright